internal/check/not_owned_file.go

```diff
@@ -167,10 +171,12 @@ func (c *NotOwnedFile) GitResetCurrentBranch(repoDir string) error {
 	return nil
 }
 
-func (c *NotOwnedFile) GitListFiles(repoDir string) (string, error) {
+func (c *NotOwnedFile) GitListFileTree(repoDir string) (string, error) {
+	args := append([]string{"ls-tree", "-r", "--name-only", "HEAD"}, c.subDirectories...)
```

Comment:
not sure how stateful the author wants this struct

maybe something like this
```suggestion
func (c *NotOwnedFile) GitListFileTree(repoDir string, subDirs ...string) (string, error) {
	args := []string{"ls-tree", "-r", "--name-only", "HEAD"}
	args = append(args, subDirs...)
```